tests/ga/test_update.py

```diff
@@ -1501,13 +1501,21 @@ def test_update_happens_when_extensions_disabled(self, _):
         self.update_handler._upgrade_available = Mock(return_value=True)
         self._test_run(invocations=0, calls=0, enable_updates=True, sleep_interval=(300,))
 
-    @patch('azurelinuxagent.common.conf.get_extensions_enabled', return_value=False)
-    def test_interval_changes_when_extensions_disabled(self, _):
+    def test_interval_should_be_default_when_extensions_enabled(self):
         """
         When extension processing is disabled, the goal state interval should be larger.
         """
-        self.update_handler._upgrade_available = Mock(return_value=False)
-        self._test_run(invocations=1, calls=1, sleep_interval=(300,))
+        with patch('azurelinuxagent.common.conf.get_extensions_enabled', return_value=True):
+            update_handler = get_update_handler()
+            self.assertEqual(6, update_handler._goal_state_period, "Incorrect goal state period when extensions are enabled")
```

Comment:
Could this reference a constant variable instead of a literal? perhaps just call the `conf` module itself?

Reply:
this test should actually be checking that we are using the default value (which happens to be 6), and the one below should verify that we are using the constant defined in update.py, but I am not planning on changing those existing tests as part of this work item. i noticed an important test case and added it here, not planning any additional improvements

Reply:
(I need to update the comment describing the new test, though, so I'll do that now

Reply:
(updated comment)